src/applications/mhv-medications/containers/Prescriptions.jsx

```diff
@@ -476,23 +477,30 @@ const Prescriptions = () => {
           <br />
           <br />
           {prescriptionsApiError ? (
-            <ApiErrorNotification errorType="access" content="medications" />
+            <>
+              <ApiErrorNotification errorType="access" content="medications" />
+              <CernerFacilityAlert className="vads-u-margin-top--2" />
+            </>
           ) : (
             <>
               {paginatedPrescriptionsList &&
               paginatedPrescriptionsList.length === 0 ? (
-                <div className="vads-u-background-color--gray-lightest vads-u-padding-y--2 vads-u-padding-x--3 vads-u-border-color">
-                  <h2 className="vads-u-margin--0">
-                    You don’t have any VA prescriptions or medication records
-                  </h2>
-                  <p className="vads-u-margin-y--3">
-                    If you need a prescription or you want to tell us about a
-                    medication you’re taking, tell your care team at your next
-                    appointment.
-                  </p>
-                </div>
+                <>
+                  <CernerFacilityAlert />
+                  <div className="vads-u-background-color--gray-lightest vads-u-padding-y--2 vads-u-padding-x--3 vads-u-border-color">
+                    <h2 className="vads-u-margin--0">
+                      You don’t have any VA prescriptions or medication records
+                    </h2>
+                    <p className="vads-u-margin-y--3">
+                      If you need a prescription or you want to tell us about a
+                      medication you’re taking, tell your care team at your next
+                      appointment.
+                    </p>
+                  </div>
+                </>
               ) : (
                 <>
+                  <CernerFacilityAlert />
```

Comment:
does it make sense to move `<CernerFacilityAlert />` outside of the IF? to not repeat it

Reply:
@vmenshutin-bylight Good point! I missed that.